### PR TITLE
Document workaround for image download

### LINKED ISSUE #694: Can't save an image in the Kiwix extension
You can't save an image from a Wikipedia article in the Kiwix web browser **extension**.
After clicking on the **Save Image line**, absolutely nothing happens:
Kiwix Save Image

### PR DESCRIPTION
**Pull request by mossroy**
Fixes #694

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -209,6 +209,17 @@ <h3 id="searchSyntax">Title search usage</h3>
                     </p>
                     <p style="text-align: right"><a href="#contents">↑ Back to Contents</a></p>
                     
+                    <h3 id="imageDownload">Image download</h3>
+                    <p>
+                        It's unfortunately not possible to download an image (from a page of a ZIM file) with the standard feature of browsers (usually right-click -> "Save
```
Comment (Jaifroid) [inline www/index.html:214]: Very slight modifications for style:

> Unfortunately it is not possible to download an image (from an article in a ZIM archive) using the standard method in most browsers (usually right-click -> "Save image as...").

File: www/index.html [hunk 2]
```diff
@@ -209,6 +209,17 @@ <h3 id="searchSyntax">Title search usage</h3>
...
+                        image as..."). See <a href="https://github.com/kiwix/kiwix-js/issues/694">issue 694</a>.
+                    </p>
+                    <p>
+                        As a workaround, it's possible to choose "Copy Image" in the context menu (after the right-click), then paste it in any suitable software (Paint, Gimp
+                        etc, or even a Word processor), then save it.
```
Comment (Jaifroid) [inline www/index.html:219]: Slight revision to indicate that the workaround is for browser extensions (e.g. right-click doesn't work in UWP apps):

> One workaround (at least in browser extensions) is to choose "Copy Image" in the context menu (after right-click or long-press)...

**Comment by Jaifroid** [comment]
@mossroy I've made some suggestions for slight revision, but otherwise this looks great. I wondered if you thought this section merits an entry in the Contents at the top of the page?

**Review by Jaifroid — APPROVED** [review]

**Comment by mossroy** [comment]
Thanks for your suggestions, I also added it in Contents.

**Pull request merged**